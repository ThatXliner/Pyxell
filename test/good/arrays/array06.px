
print [0, 0.1]
print [-1, -2.3, 4]
print ["xy", 'z']
