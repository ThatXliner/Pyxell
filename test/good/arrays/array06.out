[0.0, 0.1]
[-1.0, -2.3, 4.0]
[xy, z]
